### PR TITLE
EOS-26572: Updated cluster.yaml with latest s3service mapping

### PR DESCRIPTION
**Pull request by sachitanands**
""" Storage"""
- io - s3server,haproxy
- auth - authserver
- bg_consumer - s3bgworker
""" Control"""
- bg_producer - s3bgschedular
- openldap (ignore)

# Problem Statement
- Problem statement

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: conf/cluster.yaml.sample [hunk 1]
```diff
@@ -11,9 +11,9 @@ cluster:
       - name: hare
       - name: s3
         services:
-        - io
-        - auth
-        - bg_consumer
+        - s3server, haproxy
```
Comment (nitin-seagate) [inline conf/cluster.yaml.sample:14]: You cannot have comma separated values... Please add new entry into the list

File: conf/cluster.yaml.sample [hunk 2]
```diff
@@ -39,7 +39,7 @@ cluster:
         - fsm
       - name: s3
         services:
-        - bg_producer
+        - s3bgschedular
```
Comment (nitin-seagate) [inline conf/cluster.yaml.sample:42]: Why using the service name with prefix "s3" when it is already marked under component "s3"
Consider using name such as "bg_scheduler" or "bgscheduler"
Guess the name "schedular" is incorrect...

File: conf/cluster.yaml.sample [hunk 3]
```diff
@@ -11,9 +11,9 @@ cluster:
       - name: hare
       - name: s3
         services:
-        - io
-        - auth
-        - bg_consumer
+        - s3server, haproxy
+        - authserver
+        - s3bgworker
```
Comment (nitin-seagate) [inline conf/cluster.yaml.sample:16]: Why using the service name with prefix "s3" when it is already marked under component "s3"
Consider using name such as "bg_worker" or "bgworker"

**Review by nitin-seagate — CHANGES_REQUESTED** [review]
Why are we only changing in sample files and not in actual provisioner code, confluence etc...

**Review by ujjwalpl — APPROVED** [review]

File: conf/cluster.yaml.sample (hunk 2 above)
Comment (ujjwalpl) [reply conf/cluster.yaml.sample:42]: May be bghandler is one possible name.

**Review by ujjwalpl — APPROVED** [review]

**Review by nitin-seagate — CHANGES_REQUESTED** [review]
@sachitanands @nileshgovande Please share ticket details

File: conf/cluster.yaml.sample (hunk 3 above)
Comment (sachitanands) [reply conf/cluster.yaml.sample:16]: @nileshgovande  , @vimalraghwani20  ^^

File: conf/cluster.yaml.sample (hunk 1 above)
Comment (sachitanands) [reply conf/cluster.yaml.sample:14]: sure. will update this .

File: conf/cluster.yaml.sample (hunk 3 above)
Comment (nileshgovande) [reply conf/cluster.yaml.sample:16]: lets call it bgworker

File: conf/cluster.yaml.sample (hunk 2 above)
Comment (nileshgovande) [reply conf/cluster.yaml.sample:42]: lets call bghandler

**Comment by sachitanands** [comment]
> @sachitanands @nileshgovande Please share ticket details

Hi @nitin-seagate  pls refer below tickets for service name changes,
https://jts.seagate.com/browse/EOS-26572

**Comment by nileshgovande** [comment]
> Why are we only changing in sample files and not in actual provisioner code, confluence etc...

I assume these are 'values' and should not impact 'provisioner code' - We will need ur help if provisioner code needs changes
We will update confluence once it is proven with 'provisioner tets framework' - it is on our TODO list 
Tagging @ujjwalpl

File: conf/cluster.yaml.sample (hunk 1 above)
Comment (sachitanands) [reply conf/cluster.yaml.sample]: done.

File: conf/cluster.yaml.sample (hunk 2 above)
Comment (nitin-seagate) [reply conf/cluster.yaml.sample]: Fix typo, it should be `bgscheduler`

**Comment by nileshgovande** [comment]
Hello @nitin-seagate 

Can you please help me update below page (as I do not have access rights)

ConfigMap Confluence:
https://seagate-systems.atlassian.net/wiki/spaces/PUB/pages/611615649/CORTX+Configuration+Parameters
 
I am also adding the names of the services in POD definition files for the startup script as well

s3 currently supports both old names as well as new names for old deployment (2 PODs) as well as redefined deployment where motr and s3 are separate

**Comment by sachitanands** [comment]
changes are handled in another PR ..hence this is not required..marking it as closed.

**Pull request closed without merging**